widgetastic_manageiq/__init__.py

```diff
@@ -4274,12 +4274,37 @@ def schedule_migration(self, plan_name, cancel=False, after_mins=1):
         try:
             el = self._get_plan_element(plan_name)
             schedule_button = self.browser.element(self.ITEM_SCHEDULE_BUTTON_LOCATOR, parent=el)
+            wait_for(
+                lambda: schedule_button.is_enabled(),
+                delay=5,
+                num_sec=30,
+                message="waiting for schedule button to be enabled.",
+            )
             schedule_button.click()
             if schedule_button.text == "Schedule" and not cancel:
                 schedule_time_input = self.root_browser.element(self.ITEM_SCHEDULE_INPUT_LOCATOR)
                 schedule_time_input.click()
-                for i in range(after_mins):
-                    self.root_browser.click(self.ITEM_MODAL_MINUTE_INCREMENT_LOCATOR)
+                try:
+                    for i in range(after_mins):
+                        self.root_browser.click(self.ITEM_MODAL_MINUTE_INCREMENT_LOCATOR)
+                except NoSuchElementException:  # Throws this in FF browser
+                    self.logger.info(
+                        "Schedule using dateTimePicker failed, using datetime and timedelta."
+                    )
+                    from datetime import datetime, timedelta
```

Comment:
Required:
Remove this import and put in a header as we already imported `date` from `datetime`  lib.

Reply:
Ok, I can do that. I was thinking that we will not import this unless we need to use it. hence put it in the code there instead of top. Do you still want me to put it on top?

Reply:
Yes, It should at import only. Datetime already imported so better to import in same.